src/shared/components/SharePost.tsx

```diff
@@ -57,19 +67,38 @@ const SharePost = ({ profilePic, sx, onClick, isPost }: SharePostProps) => {
             src={profilePic}
             sx={{ width: 52, height: 52, mr: 3 }}
           />
-          <InputBase
-            name="content"
-            value={content}
-            onChange={handleChange}
-            sx={{
-              color: "black",
-              width: "500px",
-              backgroundColor: "#ededed",
-              paddingLeft: "30px",
-              borderRadius: "30px",
-            }}
-            placeholder={isPost ? "What's on your mind..." : "Write a comment"}
-          />
+          <Box width={"100%"}>
+            <InputBase
+              fullWidth
+              name="content"
+              value={content}
+              onChange={handleChange}
+              sx={{
+                color: "black",
+                backgroundColor: "#ededed",
+                paddingLeft: "30px",
+                borderRadius: "30px",
+              }}
+              placeholder={"What's on your mind..."}
```

Comment:
remove curly braces if you're rendering a string

Reply:
gotcha